extended/src/main/java/io/kubernetes/client/extended/informer/FilterField.java

```diff
@@ -0,0 +1,48 @@
+package io.kubernetes.client.extended.informer;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class FilterField {
+
+  // the name of the field
```

Comment:
delete redundant comments. (here and below)

Reply:
delete now